.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['14', '12', '10', '8']
```

Comment:
Minor: is there a specific reason for Node.js 8? According to https://nodejs.org/en/about/releases/ people are encouraged to run production code on LTS/Active releases; Node.js 8 is neither active nor maintained.

Reply:
I totally agree with this point, 
I'd also add that since we _transpile_ ramda, there isn't that much value in making sure you can run a build over multiple node versions...

This was more a porting, so that's why I kept node 8...

@CrossEye what's your view on riding of node 8 here?

Reply:
Part of why we test old versions of Node is that we don't have browser tests. At least for now we are committed to supporting ES5 environments, and so we were using older versions of Node as a proxy for that. I don't know how relevant that is at this point in time.

Reply:
It's probably not all that relevant.  But I think it's worth keeping until we cut away from that support.  And also, as this is meant to be a port, it seems the fewer changes, the better.

But I'm a little worried about code-climate stopping the build.  It might be worth doing, fixing what it notices right now and then keeping it as part of our regular commit process.  I'm much less worried about the errors shown -- we really should fix them -- than about a coverage report stopping our build.  I've been burned by that before, sometimes by removing one or two buggy tests I can no longer build; I'd rather not have that here.

Reply:
for now none of the checks are required to pass,
this is a configuration setting and can be done at another time...

![image](https://user-images.githubusercontent.com/5674833/83605960-ba601280-a570-11ea-947a-b07a9c5b69de.png)

____

I am okay to leave node 8 and plan for a removal after 1.0 is out